lib/tasks/log_variable_definitions.rake

```diff
@@ -0,0 +1,10 @@
+namespace :data_import do
+  desc "Add CsvVariableDefinition records for each file in the specified directory"
+  task :add_variable_definitions, [:path] => :environment do |_task, args|
+    CsvVariableDefinition.delete_all
+    path = Rails.root.join(args[:path])
+    service = Imports::VariableDefinitionsService.new(path:)
```

Comment:
Could we add some tests for this task and the service?

Reply:
Added